addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,219 @@
+@use '../../variables/sizing';
+@use '../../utilities/type';
+@use '../../variables/typography';
+
+$l: sizing.rems(l); // 16
+$m: sizing.rems(m); // 16
+$size-xs: sizing.rems(xs); // 8
+$size-xxs: sizing.rems(xxs); // 4
+$size-xxxs: sizing.rems(xxxs); // 2
+$size-xxxxs: sizing.rems(xxxxs); // 1
+
+.rose-form-radio-card-wide {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: ($m) * 23;
```

Comment:
Unless there is a justification for this I will recommend to never use fixed numbers i.e `23` but doing math between the already defined sizes.

I will change `width: ($m) * 23;` to `width: sizing.rems(m) * (sizing.rems(l) - sizing.rems(xxxxs))` or to be honest I will directly `width: sizing.rems(m) * sizing.rems(l)`

Reply:
This is ok to do for multiplication, IMHO.  We do it elsewhere.  We're taking a size with units and multiplying by a unitless coefficient to get some arbitrary width.

The reason to do this is because the design system doesn't come with tokens for larger values.  In many cases, the design itself is the source of an arbitrary width.  This means the code is an accurate reflection of the design intent.

Reply:
I don't think this works.  
`rem*rem isn't a valid CSS value` this is what I get when I try to multiply rems based on your suggestion.

Reply:
You are right @DhariniJeeva , I got wrong a previously implementation I did operating with sizing.

As per Randy comment, then ignore mine. All good